coati/generator.py

```diff
@@ -73,32 +73,41 @@ def generate(project_name, ppt_path):
     spaces = " " * 12
     slide_tuples = '['
 
-    path_builders = os.path.join(path, 'builders/')
+    #Generate builders/ folder prior slides creation
+    _generate_path(os.path.join(path, 'builders/'))
+    log.warn('create folder %s', "./builders/")
```

Comment:
The thing with info/warn is that we only use them to have different colors
warn is yellowish, and info is green. I wanted that folders use a different color :/